### DISCUSSION TITLE
Feature Poll: MLflow Extension for VSCode?

### DISCUSSION THREAD

**Discussion by JHibbard** [mlflow/mlflow#7126] [Polls]
# MLflow UI on VSCode?!

### And if you're a fan of another IDE or have other integration ideas, definitely let us know in a comment below. Thank you!

https://user-images.githubusercontent.com/17039389/194884069-a404ef9e-b886-40c1-8dea-39b6b73934a2.mov

**Comment by jmeisele** [comment]
What help is needed to make this happen?

**Comment by DennisWangCW** [comment]
I find it very useful to develop a vscode extension for mlflow.  Has the current mlflow version supported this function yet?

**Comment by svnv-svsv-jm** [comment]
This not existing is the reason why I use `TensorBoard`.

I'm tired of having to set up a container manually and see it from the "Simple Browser" VSCode tab.

**Comment by jacktang** [comment]
Any progress on the plugin?
